rest-api/src/main/java/io/quarkus/qe/RepositoryResource.java

```diff
@@ -36,6 +31,15 @@ public Response get(@PathParam("id") Long id) {
         }
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/")
+    public Response getAll(@QueryParam("from") @DefaultValue("0") int from, @QueryParam("to") @DefaultValue("20") int to) {
+        var allRepositories = repositoryService.findAll(from, to);
+        return (allRepositories.size() == 0) ? Response.status(Status.NO_CONTENT).build()
```

Comment:
I think it's more readable to use allRepositories.isEmpty() here.
Also, we could refactor the Response.status(X).build() which is used also in the methods get and add in this class.